chart/apisix/values.yaml

```diff
@@ -188,5 +190,5 @@ nodeSelector: {}
 
 tolerations: []
 
-affinity: {}
-
+affinity: 
```

Comment:
break change? how to custom affinity?

Reply:
cc @tokers

Reply:
Is anyone concerned about this issue? cc @gxthrj @tokers @membphis 

In addition, `affinity.enabled` is invalid for now, https://github.com/apache/apisix-helm-chart/pull/20/files#diff-9a91f0c4f242a1392081ea0f59c672ef054db7ddfe3a9f8fe98e6ebd9b06025aR100

I suggest we can roll back first.

Reply:
more related: https://github.com/apache/apisix-helm-chart/pull/26#discussion_r569105495